wandb/sdk/artifacts/artifact.py

```diff
@@ -501,6 +482,57 @@ def metadata(self, metadata: dict) -> None:
         """
         self._metadata = self._normalize_metadata(metadata)
 
+    @property
+    def ttl(self) -> Union[datetime.timedelta, None]:
+        """Artifact TTL(time to live).
+
+        Set a TTL to mark the artifact for deletion after the given duration from Artifact creation has passed.
+        To turn off previously set artifact TTL, reset TTL to wandb.ArtifactTTL.DEFAULT.
+
+        Examples:
+            Basic usage
+            ```
+            artifact.ttl = timedelta(days=10) # artifact will be marked for deletion on artifact.created_at + 10 days
+            artifact.save() # or log the artifact if its new
+            ```
```

Comment:
```suggestion
        """Time To Live (TTL).

        The artifact will be deleted shortly after TTL since its creation. None means the artifact will never expire. If TTL is not set on an artifact, it will be set to the default for its collection.
```